logger/log_writter.go

```diff
@@ -0,0 +1,41 @@
+package logger
```

Comment:
I hate to be that guy but the filename is still `log_writter.go` :grimacing:

Reply:
I edited it in github since apparently I can do that. Merging....

Reply:
oh I am sorry I thought that the file had correct name 🤦 thanks @lindgrenj6 !